src/storage-preview/README.md

```diff
@@ -262,4 +262,51 @@ az storage account network-rule remove \
     --account-name mystorageaccount
 ```
 
+#### Storage account file service properties
+##### Enable soft delete policy and set delete retention days to 100 for file service
+```
+az storage account file-service-properties update \
+    --enable-delete-retention true \
+    --delete-retention-days 100 \
+    -n mystorageaccount \
+    -g MyResourceGroup
+```
+
+##### Disable soft delete policy for file service
+```
+az storage account file-service-properties update \
+    --enable-delete-retention false \
+    -n mystorageaccount \
+    -g MyResourceGroup
+```
+
+##### Prepare for SMB multichannel
+###### Install private package
+
+`az extension add -s https://zuhdefault.blob.core.windows.net/cliext/storage_preview-0.6.1-py2.py3-none-any.whl`
```

Comment:
Should this be a public package after release?

Reply:
good point

Reply:
updated